pkg/deploy/gateway/oauth_proxy.go

```diff
@@ -147,6 +156,22 @@ func skipAuthConfig(instance *chev2.CheCluster) string {
 	return ""
 }
 
+func accessTokenConfig(instance *chev2.CheCluster) string {
+	if utils.IsAccessTokenToPass(instance) {
```

Comment:
when you have traefik config changes ready, will this become `if IsAccessTokenToPass { pass_access_token = true } else { pass_authorization_header = true }`?

I would like to avoid having both tokens in headers, when it is not needed.

Reply:
done